### PR TITLE
Document CALL IN TRANSACTIONS restrictions.

### PR DESCRIPTION
**Pull request by sherfert**

### CODE REVIEW

**Comment by sherfert** [comment]
[Document CALL IN TX restrictions](https://trello.com/c/owIwacFd/1067-document-call-in-tx-restrictions)

**Review by inmost-light — APPROVED** [review]

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala [hunk 1]
```diff
@@ -394,6 +394,13 @@ class CallSubqueryTest extends DocumentingTest {
           databaseStateBehavior = ClearState,
         ) { resultTable() }
       }
+      section("Restrictions") {
+        p("There are restrictions on a query that uses `CALL { ... } IN TRANSACTIONS`:")
```
Comment (lidiazuin) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala]:
```suggestion
        p("These are the restrictions on queries that use `CALL { ... } IN TRANSACTIONS`:")
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala [hunk 2]
```diff
@@ -394,6 +394,13 @@ class CallSubqueryTest extends DocumentingTest {
...
+        p("""* A nested `CALL { ... } IN TRANSACTIONS` inside a `CALL { ... }` clause is not supported.
+            #* A `CALL { ... } IN TRANSACTIONS` in a `UNION` is not supported.
+            #* A `CALL { ... } IN TRANSACTIONS` after a write clause is not supported, unless that write clause is inside a `CALL { ... } IN TRANSACTIONS`.""".stripMargin('#'))
```
Comment (lidiazuin) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala:401]:
```suggestion
            #* A `CALL { ... } IN TRANSACTIONS` after a write clause is not supported, unless that write clause is inside a `CALL { ... } IN TRANSACTIONS`.""".stripMargin('#')).
```

Comment (sherfert) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala:401]: There is already a period after that sentence, just before the triple quotation mark.

Comment (lidiazuin) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala:401]: You are right! Please ignore this one then :)

**Comment by lidiazuin** [comment]
@sherfert your PR says 4.4 but it was created and merged against dev branch. If you don't mind I can try to fix this.

**Pull request merged**